dedupe/dedupe_test.go

```diff
@@ -26,18 +25,52 @@ func TestPacketDeduplicatorSeen(t *testing.T) {
 	}
 
 	// Wait for expiration
-	time.Sleep(expiresAfter + 100*time.Millisecond)
+	time.Sleep(expiresAfter + time.Millisecond)
 
 	// Test Seen with same packetID after expiration
 	if dedup.Seen(sender, packetID) {
 		t.Error("Expected the packet to not have been seen after expiration")
 	}
 }
 
+func TestDuplicatorProp(t *testing.T) {
```

Comment:
Go's built-in property test stuff is pretty awful, but still better than nothing, IMO.  There are external libs that do it better (probably).